package.json

```diff
@@ -35,7 +35,6 @@
     "@ember/render-modifiers": "^1.0.2",
     "@glimmer/component": "^1.0.0",
     "@glimmer/tracking": "^1.0.0",
-    "@kanselarij-vlaanderen/au-kaleidos-css": "^1.4.2",
```

Comment:
Ik vermoed dat dit toch een `devDependency` is, gezien we de `scss` files hieruit (samen met andere in-repo `scss`) gaan builden tot 1 `css`-file om tot bij de eindgebruiker te brengen.